### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Upgrade to v.0.12.1

[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/lutim_ynh%20PR53%20(ericgaspar)/)

### CODE REVIEW

File: scripts/backup [hunk 1]
```diff
@@ -19,55 +19,51 @@ ynh_abort_if_errors
...
 
 #=================================================
 # BACKUP NGINX CONFIGURATION
 #=================================================
-ynh_script_progression --message="Backing up nginx web server configuration..."
 
 ynh_backup --src_path="/etc/nginx/conf.d/$domain.d/$app.conf"
 
-#=================================================
-# SPECIFIC BACKUP
```
Comment (yalh76) [inline scripts/backup:47]: you should keep that

File: scripts/backup [hunk 2]
```diff
@@ -19,55 +19,51 @@ ynh_abort_if_errors
...
-ynh_script_progression --message="Backing up cron file"
 
 ynh_backup --src_path="/etc/cron.d/$app"
 
 #=================================================
 # END OF SCRIPT
 #=================================================
 
-ynh_script_progression --message="Backup script completed for $app. (YunoHost will then actually copy those files to the archive)." --last
+ynh_print_info --message="Backup script completed for Lutim. (YunoHost will then actually copy those files to the archive)."
```
Comment (yalh76) [inline scripts/backup:69]: why switching $app to the app name ?

File: scripts/change_url [hunk 3]
```diff
@@ -163,4 +163,4 @@ ynh_maintenance_mode_OFF
 # END OF SCRIPT
 #=================================================
 
-ynh_script_progression --message="Change of URL completed for $app" --last
+ynh_script_progression --message="Change of URL completed for Lutim" --last
```
Comment (yalh76) [inline scripts/change_url:166]: why switching $app to the app name ?

File: scripts/install [hunk 4]
```diff
@@ -264,4 +262,4 @@ ynh_send_readme_to_admin --app_message="mail_to_send" --recipients="root" --type
 # END OF SCRIPT
 #=================================================
 
-ynh_script_progression --message="Installation of $app completed" --last
+ynh_script_progression --message="Installation of Lutim completed" --last
```
Comment (yalh76) [inline scripts/install:265]: why switching $app to the app name ?

File: scripts/remove [hunk 5]
```diff
@@ -36,7 +36,7 @@ ynh_remove_systemd_config
 # Check if the service is declared in YunoHost
 if ynh_exec_fully_quiet yunohost service status $app
 then
-	ynh_script_progression --message="Removing $app service..."
+	ynh_script_progression --message="Removing Lutim service..."
```
Comment (yalh76) [inline scripts/remove:39]: Same here for `$app`

**Review by yalh76 — CHANGES_REQUESTED** [review]
Keep the package the most generic will help to maintain it on the long run.

File: scripts/backup (hunk 2 above)
Comment (ericgaspar) [reply scripts/backup:71]: @yalh76 Thanks for reviewing this PR!
I wanted to keep capitalization of the name in messages...

Comment (yalh76) [reply scripts/backup:71]: :/

**Review by yalh76 — APPROVED** [review]

**Pull request merged**